frontend/components/clients/ClientArticle.tsx

```diff
@@ -18,11 +17,10 @@ const ClientArticle: React.FC<ClientArticleProps> = ({
     <div
       key={client.id}
       className={classNames(
-        "flex justify-center items-center flex-row space-y-5",
-        "w-7/12"
+        "sm:block w-7/12 md:w-7/12 lg:flex space-y-5 m-auto my-5 pt-10"
       )}
     >
-      <Wrapper className="bg-gray-100 mr-8">
+      <Wrapper className="bg-gray-100 mr-0 lg:mr-8 min-h-[200px] md:flex-none lg:w-5/12 m-0">
```

Comment:
```suggestion
      {client.logo?.url ? (
        <Wrapper className="bg-gray-100 mr-0 lg:mr-8 min-h-[200px] md:flex-none lg:w-5/12 m-0">
          <img
            src={`${LOCAL_BACKEND_URL}${client.logo?.url}`}
            alt={`${client.name} Logo`}
            className={classNames("w-20 m-4")}
          />
        </Wrapper>
      ) : null}
```

Client logo is not required, and should not be rendered if it is no image is uploaded. However, it would look kinda silly with some preview articles with images and some without... 🤔